README.md

```diff
@@ -3,6 +3,7 @@
 [![Tests](https://github.com/quantumjot/BayesianTracker/actions/workflows/build.yml/badge.svg)](quantumjot/BayesianTracker/actions)
```

Comment:
Do you need to change this since you renamed the workflow?

Reply:
Ooh yes, thanks for catching!

Reply:
np - I was just trying to work out what wizardy was required to get this to work!

Reply:
The actual fix was putting `envsitepackagesdir` into `--cov={envsitepackagesdir}/btrack`, so `pytest-cov` is running coverage against the package installed in the tox environment, not the local checkout of the package.